pubtools/_ami/tasks/push.py

```diff
@@ -109,6 +109,7 @@ def upload(self, push_item):
             "virt_type": push_item.virtualization,
             "root_device_name": push_item.root_device,
             "volume_type": push_item.volume,
+            "uefi_support": push_item.uefi_support,
```

Comment:
In requirements.in you'll have to bump the minimum versions of both cloudimg and pushsource to the versions supporting this field.